src/rpc/blockchain.cpp

```diff
@@ -439,6 +439,11 @@ static RPCHelpMan getblockfrompeer()
         },
         [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
 {
+    RPCTypeCheck(request.params, {
+        UniValue::VSTR, // blockhash
+        UniValue::VNUM, // peer_id
+    }, true);
```

Comment:
nit: This RPC does not  accept null params.

```suggestion
    RPCTypeCheck(request.params, {
        UniValue::VSTR, // blockhash
        UniValue::VNUM, // peer_id
    });
```